libraries/lib-ffmpeg-support/wrappers/AVPacketWrapper.cpp

```diff
@@ -31,6 +31,15 @@ AVPacketWrapper::~AVPacketWrapper()
 {
    if (mFFmpeg != nullptr && mAVPacket != nullptr)
    {
+      // Must we be sure not to leak packet side data when av_packet_free
+      // is not defined?
+
+      // What old code did, but deprecated:
+      // mFFmpeg->av_free_packet(mAVPacket);
+
+      // Favored way:
+      mFFmpeg->av_packet_unref(mAVPacket);
```

Comment:
I will move it down to the case where av_free is used. There is no need for this call when av_packet_free is used for sure